torch/csrc/jit/mobile/parse_bytecode.cpp

```diff
@@ -108,6 +112,42 @@ void parseInstructions(
     OpCode op_code = opCodeCache.parse(*ins_item[0].toString());
     int X = ins_item[1].toInt();
     int N = ins_item[2].toInt();
+
```

Comment:
I would recommend to pull this functionality as a standalone function that operates on mobile::Function or mobile::Code. 

So you are not "putting upgrader in place while parsing" but first do parsing and get mobile::Function; then apply upgrader on top.

Reply:
Chatted offline. It's better to have a separate function so it's easier to apply when we switch to flatbuffer. Address in the latest commit. Performance might be slightly affected. Consider we just separate out the operation on another loop, it will be neutral.

Reply:
Address in the latest commit.